apps/bandwagon/tests/test_views.py

```diff
@@ -928,8 +928,8 @@ def test_remove_nonexistent(self):
     def test_no_ajax_response(self):
         r = self.client.post(self.add, {'addon_id': self.addon.id},
                              follow=True)
-        self.assertRedirects(r, reverse('collections.detail',
-                                        args=['jbalogh', 'mobile']))
+        self.assert3xx(r, reverse('collections.detail',
+                                  args=['jbalogh', 'mobile']))
```

Comment:
I'm not really a fan of this kind of indentation, I'd rather see something like

```
self.assert3xx(r,
    reverse(...))
```

Though we can discuss that and since we don't have different styles everywhere I'm fine with ignoring it for now.